### PR TITLE
Robots convert damage to clone under a threshold

### PR DESCRIPTION
**Pull request by tanker4390**
## About The Pull Request
Combat robots, while under `-50` health, will convert BRUTE and BURN damage to clone damage at a rate of  `1` every `30` seconds, to a maximum of `50` clone damage. They will receive scary messages to let them know when this is happening.
## Why It's Good For The Game
Currently, combat robots can take plenty of punishment, heal back, and not have to limp back to medbay unless they die or take high burst damage. Robots will be encouraged to stay healthy and not overclock more than necessary (or at all... even more than before), and xenos will feel more like they're doing lasting damage against the metal blitzkrieg.
## Changelog
:cl: Tanker
balance: Combat robots now slowly convert damage to clone damage while under a certain health threshold.
/:cl:

### CODE REVIEW

**Comment by Barnet2** [comment]
This is actually a fairly interesting idea that I haven't thought of before. Might be worth a try to give robots a form of attrition outside of draining (which is hard to do given no crit drag.. or crit for that matter)

**Comment by Arctrooper8** [comment]
I like the fluff text for the warning message, it's very flavorful for robots.

**Comment by technicianman** [comment]
Robots already lack critical passive heal capabilities (apart from dispenser but that has a whole set of probloms) And there are PLENTY of cases where you don't have time to autorepair or to weld/cable yourself. All this would you is punish robots more for not having any passive regen capabilities. Humans can already be ridiculously tanky with premed and especially if they use rr considering that a full heal is only 10 cl.

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -465,6 +465,9 @@
 	H.voice_filter = initial(H.voice_filter)
 	H.health_threshold_crit = -50
 
+#define CLONELOSS_THRESHOLD 0
```
Comment (Xander3359) [inline code/modules/mob/living/carbon/human/species.dm:468]: This should probably be the same threshold as a human being in crit (-50).
While I like the concept, punishing robots for being at 0 hp, when a human is perfectly capable of existing at 0 hp is a bit too much.
The idea being that if you are at/under critical threshold for too long, your chassis will degrade to the point that you will die from it.

**Comment by ivanmixo** [comment]
Meh not really a fan of this. Their cloneloss is already harder to take care of compared to humans so getting drained hurts more, feels unnecesary to have even more on top.

**Comment by Coastfront** [comment]
Combat robots:
Cannot heal passively
Are obscenely slow compared to humans
And their only forms of healing, are both easily interruptable via flankers, as well as still being slower than marine healing.

Combat robots advantages are thus:
Stronger on defense, where its harder to flank them
Can heal theoretically infinitely and only experience attrition via resources.
Get to ignore defilers/fractures.

All this would do is encourage combat robots to play in both the, already most effective way to play them, and the objective least fun way to play tgmc.

You attempt to amend a burnt, charred hand with the scars of a knife's edge on their fingers: You're only feeding into a worse injury.
Your why its good for the game speaks of somebodies prison sentence, with none of why such tresspass has been made. Combat robots shrug off damage, and are quite durable; Penalized with anchored steps. Marines have to limp back to medbay because marines can, in the actual heat of combat, take more abuse before disengaging, and can actually capitalize on injured xenos via chasing them. Combat robots literally just have to watch their opponent walk away from them if they didn't stunlock them into death.

You speak only of plucking a weeds roots via planting another to choke it. This pr creates a worse issue than it solves, further still solves what is barely an issue.

**Comment by coldironwarrior** [comment]
>converting critical damage to 50 cloneloss on the marines that can't passively heal to prevent continued existence at sub-critical thresholds, can't sleep CL off, and can't utilize somnolent to reduce CL

this is clown shit 🤡

**Comment by Coastfront** [comment]
>While I’d argue humans are better than robots in both killing and survivability (being able to both run to skidaddle out of fights or medding up to last longer), I think this mechanic is pretty cool+unique and fits with the theme of TGMC and combat robots as a whole. I like unique mechanics separating the two species more.

It pains me so, that from the same tongue could acknowledgement fall does still pointlessness reside.
Combat robots, are, yes, objectively worse than marines in just about every relevant aspect, and are only made unique via their lack of attrition.

So why, in our labors to drag a mountain from the sea, do we erode away its peak? Why must their "unique and interesting mechanics" be "gets fucking cloneloss for trying to use their larger healthbar" 

Already must they act as a sea buoy, helpless to watch the fish in the waters, literally unable to get kills on any remotely competent/mobile xeno if they don't have humans nearby to capitalize on damage they deal, so why must we punish them further?
This literally just discourages robots from extending, because if they dont play it absurdly safe, due to their laughably slow movement speed, just about any xeno gets to have their way with them if they even slightly over-extend; marines ailed the same shrug it off passively, only needing a splint if they over-extend near double what a robot could afford.
While then, under this, robot slightly over-extends, get fucked, you have now near permanently lost part of your healthbar.


It needs not distinction by frailty, there is literally a fucking ocean of more interesting ideas anybody could use in an effort to add more identity to combat robots beyond "slow human without attrition or chem healing"

**Comment by DerpiestBruh** [comment]
The punishment to overextending Robots is already having to spend an absurd amount of time to fix themselves without cooperation and how vulnerable they are during that time. 50CL seems a bit too extreme for taking advantage of the very few pros you have.

**Comment by Coastfront** [comment]
> > The punishment to overextending Robots is already having to spend an absurd amount of time to fix themselves without cooperation and how vulnerable they are during that time. 50CL seems a bit too extreme for taking advantage of the very few pros you have.
> 
> you all need to read the PR. it would take 12.5 minutes of staying as negative health to hit 50CL. If you can do that as combat robot while staying alive, you deserve it
> 
> edit: PR changed, its now 25 minutes of being at -50

The complaint, is that this is fucking stupid.
Why do we allow, even extremely incremental nerfs? This is the jab that leads into a punch. This nerf will get past because its "Only x amount"
and then that amount will be raised in the future, and by the same key will it step through the same door.

The concept is a bad idea, regardless of how it's implemented, and doesn't need to happen.
Just because somebody goes through the effort to make and pitch a PR doesn't mean we need to unilaterally fucking allow them.

**Comment by FosterGrime-Heymynameslouie** [comment]
this kinda sucks
not only do you have to sit and wait to repair (both auto and manual)
be a fat ass (slow)
never passively heal
all of the above compounding on eachother increasing the likeliness of you dying and getting cloneloss more
you also, now that you have an inrepairable damage value attached to you come up even more often while you try to play the game, have the ability to sit and wait for avenues _(medvac, tadpole, rarely alamo)_ to get cloneloss to repair! 
almost as if the combat droid, who, who knows if they were intended for combat, needed their combat ability stifled even more
thrilling!

**Comment by technicianman** [comment]
1 CL per 15 seconds from the -50 threshold is only gonna make this a annoying little thing to deal with, it wont even change the balance all its gonna do is make sure every robot has at least 5 cl, pointless?

**Comment by Sirsnek15** [comment]
I clicked on this thinking it was gonna be another miss at attempting to make robos more balanced but this is actually really cool

**Comment by zzurge** [comment]
xope PR

**Comment by TiviPlus** [comment]
why is this dnm

**Comment by ivanmixo** [comment]
Robots already have enough shit to deal with, this isn't necesarry

**Comment by Coastfront** [comment]
> why is this dnm

Writhing beneath a boot, should even the frailest weed find its roots still in soil: it remains.

Its in DNM when it should outright be closed. This pr has no decent reason to be added to the game.

**Comment by ivanmixo** [comment]
Wharever go away

**Pull request closed without merging**